### ISSUE TITLE
Logo Request: GoDaddy

### ISSUE THREAD

**Issue by ben-morin** [authenticatorplus/logos#107]
Please provide below info, all are mandatory

1. sso.godaddy.com
2. [customer] @ GoDaddy
3. 

![godaddy-logo](https://user-images.githubusercontent.com/1482423/34084915-9d6c5d04-e345-11e7-821d-da6d48e0ef14.jpg)

**Comment by MufriA** [comment]
Added, pick from custom icons list

**Comment by Caligatio** [comment]
Is this working still?  The GoDaddy icon for me is a weird wifi symbol and, when picking it, doesn't actually apply (Android).

**Comment by ixtlanian** [comment]
@MufriA as noted above, GoDaddy icon does not work. Instead of the logo, there's a WiFi symbol. Appreciate if you could fix it.

**Comment by ixtlanian** [comment]
@MufriA Please advise if you had a chance to look into this issue.

**Comment by MufriA** [comment]
Even after selecting name from custom icons list is it not working, wifi symbol means network issue.

Can you try to set again

**Comment by ixtlanian** [comment]
Whatever you did on the backend caused all the icons to be reloaded in the catalog of all services that you have. It was visible because I saw a delay in rendering the icons as I was scrolling through the list of services.
So, now GoDaddy icon is visible and working.
However a few other things happened:
1. DocuSign service appeared, but with WiFi icon. I have this as case #195 
2. Kickstarter icon, which was working fine before this changed and when I tried to re-select it, I noticed that Kickstarter is not even in the list of services now.
Please advise.

**Comment by MufriA** [comment]
I have addd Kickstarter too in the custom list, can you try now.

**Comment by ixtlanian** [comment]
1. Kickstarter service reappeared in the list, but with a WiFi icon, so something is wrong with it again.
2. DocuSign's WiFi icon has been fixed now and I can see it correctly.
@MufriA appreciate if you look into Kickstarter again.

**Comment by ben-morin** [comment]
Just as an observation, it seems to me that the last icon in the grid of images (which seems to be completely unsorted, by the way) or the last added seems to suffer from these reported issues.  When the GoDaddy icon was the last in the grid of images then it was having issues.  Now I see issues with the Kickstarter icon.  Perhaps the issue lies in code around the the rendering of the images in that grid.

In other words, if you add a new image after kickstarter, it will likely stop behaving incorrectly and the new image will take on the reported issues.  Just a theory.

**Comment by ixtlanian** [comment]
@MufriA the comment above by @ben-morin seems to align with my observations.
Can you please review it and fix the custom list?
Kickstarter is now showing WiFi icon in the custom logo list.
Sorry for posting in this issue thread, but that's where we ended up discussing it.
